vendor/engines/secure_rooms/app/presenters/secure_rooms/account_presenter.rb

```diff
@@ -0,0 +1,23 @@
+module SecureRooms
+
+  class AccountPresenter < SimpleDelegator
+
+    def self.wrap(accounts)
+      accounts.map { |account| new(account) }
+    end
+
+    def to_json
+      {
+        id: id,
+        type: type,
+        description: description,
+        account_number: account_number,
+        expiration_month: expiration_month,
+        expiration_year: expiration_year,
+        name: owner_user.try(:full_name)
```

Comment:
The only thing in this whole hash that I was surprised at was the `try`; why would there not be an owner name? Is this gonna be common?

If this is unavoidable and _not_ common: could we put something useful (so there's no weird blank space in the UI), like maybe the facility name or something?